### PR TITLE
🧹 `Furniture`: Complete the rename!

### PR DESCRIPTION
**Pull request by zspencer**
- Requires: https://github.com/zinc-collective/convene/pull/1208
- https://github.com/zinc-collective/convene/issues/709

OK, so I think this is a straight-up-rename; but there may be stuff that wants fixing...

### CODE REVIEW

File: .rubocop_todo.yml [hunk 1]
```diff
@@ -41,11 +41,11 @@ RSpec/ExampleLength:
     - 'spec/furniture/journal/entries_request_spec.rb'
     - 'spec/furniture/markdown_text_block_spec.rb'
     - 'spec/furniture/marketplace/products_controller_request_spec.rb'
-    - 'spec/models/furniture_placement_spec.rb'
+    - 'spec/models/furniture_spec.rb'
     - 'spec/models/membership_spec.rb'
     - 'spec/models/room_spec.rb'
     - 'spec/policies/room_policy_spec.rb'
-    - 'spec/requests/furniture_placements_controller_request_spec.rb'
+    - 'spec/requests/furnitures_controller_request_spec.rb'
```
Comment (anaulin) [inline .rubocop_todo.yml:48]: You could just un-lint these 😉

**Review by anaulin — APPROVED** [review]
Yes!!!!

File: .rubocop_todo.yml (hunk 1 above)
Comment (zspencer) [reply .rubocop_todo.yml:48]: I was afeared of trying to do even more in this giganormoustrosity

Comment (anaulin) [reply .rubocop_todo.yml:48]: If you put it in a separate commit I won't object!

But up to you, fine either way.

**Pull request merged**